tests/torch/data/reference_graphs/const_sparsity/ssd_mobilenet.dot

```diff
@@ -420,16 +416,12 @@ strict digraph  {
 "195 MobileNetSSD/SSDDetectionOutput[detection_head]/view_3" -> "198 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_0";
 "196 MobileNetSSD/SSDDetectionOutput[detection_head]/view_4" -> "198 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_0";
 "197 MobileNetSSD/SSDDetectionOutput[detection_head]/view_5" -> "198 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_0";
-"198 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_0" -> "208 MobileNetSSD/SSDDetectionOutput[detection_head]/view_13";
 "199 MobileNetSSD/SSDDetectionOutput[detection_head]/view_6" -> "205 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_1";
 "200 MobileNetSSD/SSDDetectionOutput[detection_head]/view_7" -> "205 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_1";
 "201 MobileNetSSD/SSDDetectionOutput[detection_head]/view_8" -> "205 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_1";
 "202 MobileNetSSD/SSDDetectionOutput[detection_head]/view_9" -> "205 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_1";
 "203 MobileNetSSD/SSDDetectionOutput[detection_head]/view_10" -> "205 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_1";
 "204 MobileNetSSD/SSDDetectionOutput[detection_head]/view_11" -> "205 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_1";
 "205 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_1" -> "206 MobileNetSSD/SSDDetectionOutput[detection_head]/view_12";
-"205 MobileNetSSD/SSDDetectionOutput[detection_head]/cat_1" -> "209 MobileNetSSD/SSDDetectionOutput[detection_head]/view_14";
 "206 MobileNetSSD/SSDDetectionOutput[detection_head]/view_12" -> "207 MobileNetSSD/SSDDetectionOutput[detection_head]/softmax_0";
-"208 MobileNetSSD/SSDDetectionOutput[detection_head]/view_13" -> "210 /nncf_model_output_0";
-"209 MobileNetSSD/SSDDetectionOutput[detection_head]/view_14" -> "211 /nncf_model_output_1";
```

Comment:
Same in SSD, where the detection output in eval mode is actually under `with no_nncf_trace()` and therefore won't be present in the graph.